doc/source/Platforms.rst

```diff
@@ -548,9 +560,9 @@ The following instructions were used to prepare a basic Ubuntu 20.04 system as i
    apt-get upgrade
 
    # Compilers
+   apt install -y gcc g++ gfortran gdb
    apt install -y gcc-10 g++-10
    apt install -y gfortran-10
-   apt install -y gdb
```

Comment:
Shouldn't we include the debugger? When you run `xcode-select install` to get the command line tools on MacOS, you get the compilers and a debugger (lldb) installed.

Reply:
This is now a few lines up (line 563 in the updated file).

Reply:
Sorry I missed that. Looks good